libcontainer/cgroups/ebpf/ebpf.go

```diff
@@ -72,6 +74,60 @@ func findAttachedCgroupDeviceFilters(dirFd int) ([]*ebpf.Program, error) {
 	return nil, errors.New("could not get complete list of CGROUP_DEVICE programs")
 }
 
+var (
+	haveBpfProgReplaceBool bool
+	haveBpfProgReplaceOnce sync.Once
+)
+
+// Loosely based on the BPF_F_REPLACE support check in
+//   <https://github.com/cilium/ebpf/blob/v0.6.0/link/syscalls.go>.
+func haveBpfProgReplace() bool {
+	haveBpfProgReplaceOnce.Do(func() {
+		prog, err := ebpf.NewProgram(&ebpf.ProgramSpec{
+			Type:    ebpf.CGroupDevice,
+			License: "MIT",
```

Comment:
Any specific reason for this?

Reply:
Ah, seems copied from https://github.com/cilium/ebpf/blob/v0.6.0/link/syscalls.go :+1:

Reply:
Yeah, we just need to pick a license and I didn't change it from the cilium one (even though it's so trivial I'm not even sure copyright applies here).